modules/bmo-validating-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,27 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="validating-the-hostfirmwaresettings-resource_{context}"]
+= Validating the HostFirmwareSettings resource
+
+After editing the `spec.settings` section of the `HostFirmwareSettings` resource, validate that the resource complies with the `FirmwareSchema` resource.
```

Comment:
So this section isn't quite how it works.  The user doesn't need to do anything to validate the resource and no value will be returned from that command. If the user entered an invalid value the baremetal operator will validate it and:
- The condition of type "Valid" will show as "False"
- An event will be generated describing the error and can be seen by using the "oc describe hfs/<host_name> -n openshift-machine-api" command at the bottom, for example
Events:
  Type    Reason            Age   From                                    Message
  ----    ------            ----  ----                                    -------
  Normal  ValidationFailed  2s    metal3-hostfirmwaresettings-controller  Invalid BIOS setting: Setting ProcTurboMode is invalid, unknown enumeration value - Foo

The user can then update the value. By viewing the FirmwareSchema "oc get firmwareschema /<schema_name_that_hfs_uses>-n openshift-machine-api -o yaml" the user can see the type and limits of the value to enter.

Reply:
I've made some updates and renamed the module, but I'm still not quite sure if it's right.